Resources/Audio/_RMC14/Voice/Avali/attributions.yml

```diff
@@ -8,8 +8,7 @@
   copyright: "Taken from Steam 'Avali sounds' mod / Starbound. Made by Steam user: https://steamcommunity.com/id/Nefuki with permission."
   source: "https://steamcommunity.com/sharedfiles/filedetails/?id=3164757879"
 
-
-- files: ["avali_laugh"]
+- files: ["avali_cough, avali_cry1, avali_cry2, avali_laugh1, avali_laugh2, avali_scream2, avali_whistle"]
   license: "CC-BY-NC-SA-3.0"
```

Comment:
You can't license these sounds unless you own them or have proof of an existing license for these sounds, pulling them from a reddit link + google drive won't meet the legal requirements